src/components/app/details/cdDetails/CDDetails.tsx

```diff
@@ -138,6 +139,7 @@ export default function CDDetails() {
             value: `${item.environmentId}`,
             label: item.environmentName,
             pipelineId: cdPipelinesMap.get(item.environmentId).id,
+            deploymentAppDeleteRequest: _deploymentAppDeleteRequest.get(item.deploymentAppDeleteRequest).deploymentAppDeleteRequest,
```

Comment:
can't we use item.deploymentAppDeleteRequest directly?

Reply:
Done